Mage.Sets/src/mage/cards/s/ShareTheSpoils.java

```diff
@@ -0,0 +1,348 @@
+        super(ownderId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{1}{R}");
+
+        // Increment everytime a new copy is cast so that they don't share
+        exileZoneIdString = this.getId().toString();
+
+        // When Share the Spoils enters the battlefield or an opponent loses the game,
+        // exile the top card of each player’s library.exile the top card of each player’s library.
+        this.addAbility(new ShareTheSpoilsExileETBAndPlayerLossAbility(exileZoneIdString));
+
+        // During each player’s turn, that player may play a land or cast a spell from among cards exiled with Share the Spoils,
+        Ability castAbility = new SimpleStaticAbility(new ShareTheSpoilsPlayExiledCardEffect(exileZoneIdString));
+        // and they may spend mana as though it were mana of any color to cast that spell.
+        castAbility.addEffect(new ShareTheSpoilsSpendAnyManaEffect(exileZoneIdString));
+        castAbility.setIdentifier(MageIdentifier.ShareTheSpoilsWatcher);
+        this.addAbility(castAbility, new ShareTheSpoilsWatcher());
+
+        // When they do, exile the top card of their library.
+        Ability exileCardWhenPlayedACard = new ShareTheSpoilsExileCardWhenPlayACardAbility(exileZoneIdString);
+        this.addAbility(exileCardWhenPlayedACard);
+    }
+
+    private ShareTheSpoils(final ShareTheSpoils card) {
+        super(card);
+        this.exileZoneIdString = card.exileZoneIdString;
+    }
+
+    @Override
+    public ShareTheSpoils copy() { return new ShareTheSpoils(this); }
+}
+
+//-- Exile from Everyone --//
+class ShareTheSpoilsExileETBAndPlayerLossAbility extends TriggeredAbilityImpl {
+
+    ShareTheSpoilsExileETBAndPlayerLossAbility(String exileZoneIdString) {
+        super(Zone.BATTLEFIELD, new ShareTheSpoilsExileCardFromEveryoneEffect(exileZoneIdString));
+    }
+
+    private ShareTheSpoilsExileETBAndPlayerLossAbility(final ShareTheSpoilsExileETBAndPlayerLossAbility ability) {
+        super(ability);
+    }
+
+    @Override
+    public boolean checkEventType(GameEvent event, Game game) {
+        return event.getType() == GameEvent.EventType.ENTERS_THE_BATTLEFIELD ||
+                event.getType() == GameEvent.EventType.LOST || // Player conceedes
+                event.getType() == GameEvent.EventType.LOSES;  // Player loses by all other means
+    }
+
+    @Override
+    public boolean checkTrigger(GameEvent event, Game game) {
+        // The card that entered the battlefield was this one
+        if (event.getType() == GameEvent.EventType.ENTERS_THE_BATTLEFIELD) {
+            return event.getTargetId().equals(getSourceId());
+        }
+
+        // A player lost the game
+        return true;
+    }
+
+    @Override
+    public TriggeredAbility copy() {
+        return new ShareTheSpoilsExileETBAndPlayerLossAbility(this);
+    }
+
+    @Override
+    public String getTriggerPhrase() {
+        return "When Share the Spoils enters the battlefield or an opponent loses the game";
```

Comment:
don't forget to use `{this}` instead of the card's name, also if you're gonna use a trigger phrase then it should end with a comma and a space. In this case you should probably just override `getRule` and put the full text there.